BellaSalon/Controllers/HomeController.cs

```diff
@@ -0,0 +1,188 @@
+﻿//Don't know how to change directory of my ALREADY submitted Checkpoint#1, 
+//so I make a new branch and also making some changes in this project by typing 
+// these comments, so I can "git add" it to a new branch.
+//
+//*********************************************************************
+//And I'm not being late.
+//
+//*********************************************************************
+
+using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.Linq;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Mvc;
+using BellaSalon.Models;
+
+namespace BellaSalon.Controllers
+{
+    public class HomeController : Controller
```

Comment:
This controller does a lot. In the future, I would recommend splitting this into different classes.